content/src/simple.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>jQuery UI Slider - Default functionality</title>
+<link rel="stylesheet" type="text/css" href="//<!--#echo var="site"-->/lib/jquery-ui.css">
+<link rel="stylesheet" type="text/css" href="//<!--#echo var="site"-->/lib/jquery-ui-slider-pips.css">
+  <link rel ="stylesheet" href = "editor.less">
+  <script src="jquery.js"></script>
+  <script src="jquery-ui.js"></script>
+  <script src = "jquery-ui-slider-pips.js"></script>
+  <script>
+ $("#circles-slider")
+    .slider({
+        max: 10,
+        value: 5
+    })
+    .slider("pips");
+  </script>
+</head>
+<body>
+<div id="circles-slider"></div>
+ 
+ 
+</body>
+</html>
```

Comment:
Both simple.html and protractor.js are extraneous files that have just been removed from this pr!